comment-mail/includes/templates/type-s/email/comment-notification/snippet/message-heading.php

```diff
@@ -4,24 +4,24 @@
 
 		[if sub_comment]
 			[if subscribed_to_own_comment]
-				New Replies to Your Comment on "[sub_post_title_clip]"
+				New Replies to Your Comment on <em>[sub_post_title_clip]</em>
 			[else]
-				New Replies to <a href="[sub_comment_url]">a Comment</a> on "[sub_post_title_clip]"
+				New Replies to <a href="[sub_comment_url]">a Comment</a> on <em>[sub_post_title_clip]</em>
 			[endif]
 		[else]
-			New Comments on "<a href="[sub_post_comments_url]">[sub_post_title_clip]</a>"
+			New Comments on <a href="[sub_post_comments_url]"><em>[sub_post_title_clip]</em></a>
```

Comment:
@renzms Can you please move the `<em></em>` markup _outside_ the anchor element on this line? You already did that everywhere else; this was the only line you did differently.